docs/chat-configuration/ui-customization/queue-bar.md

```diff
@@ -160,14 +160,15 @@ ChatUIProvider(context).apply {
 Configure cancel message
 {: .strong-sub-title}   
 
-`Cancel chat` displayed text, is customizable via the chat window customizations. You can change it's default value and add more branding languages values by setting the text for `api#chat#unavailable_email` branding key.   
+`Cancel chat` displayed text, is customizable via the chat window customizations. You can change the default value and add more branding languages values by setting the text for `api#chat#unavailable_email` branding key.   
 > The displayed text is branded according to the chat selected language.
 
 ---
 
 ## Listening to queue position changes
-Queue position changes can be listened to by a [custom UI component](#update-position-display), as described before.    
-In order to listened to position changes not from a custom UI implementation, you need to subscribe to the `Notifications.QueuePosition` notification using the `ChatController`.
+Queue position changes can be listened to by a [custom UI component](#update-position-display), as described before. 
+## I WOULD ADD A LINK OR REFERNCE.   
+to listen to position changes not from a custom UI implementation, there is a  need to subscribe to the `Notifications.QueuePosition` notification using the `ChatController`.
```

Comment:
sentence startup is missing here

Reply:
Maybe change the "to listen" to "Listening".